src/firefly/java/edu/caltech/ipac/firefly/server/persistence/QueryRTreeInventory.java

```diff
@@ -127,7 +127,7 @@ public void prepareTableMeta(TableMeta meta, List<DataType> columns, ServerReque
                 "ra1", "dec1", "ra2", "dec2", "ra3", "dec3", "ra4", "dec4"
         };
         String dataset = request.getParam("RequestedDataSet");
-        meta.setAttribute(MetaConst.DATASET_CONVERTER, dataset);
+        meta.setAttribute(MetaConst.MISSION, dataset);
```

Comment:
should here be MetaConst.MISSION or MetaConst.IMAGE_SOURCE_ID?

Reply:
should be `IMAGE_SOURCE_ID`, i will fix it.

Reply:
IMAGE_SOURCE_ID. fixed.